### DISCUSSION TITLE
Backup and restore

### DISCUSSION THREAD

**Discussion by tokubi** [karakeep-app/karakeep#599] [Q&A] [answered]
For a Hoarder running in Docker, is there any way to backup all content (including all cached content, screenshots etc.) and then restore it, for example on different hardware so that the content doesn't have to be downloaded again? Some kind of detailed manual describing the steps that need to be taken to achieve this would be great. 
Hoarder is absolutely fantastic software, however as the amount of data I store here grows I am increasingly concerned that there is no way (as far as I know) to back up and then successfully restore (transfer) this data. 
I have read https://github.com/hoarder-app/hoarder/discussions/396 but unfortunately it did not help me solve this problem.
Thank you for your help.

**Comment by kamtschatka** [comment, marked as answer]
shut down hoarder, backup the data dir and then start hoarder again.
meilisearch backup idk, but you can reindex all the data again after a backup restore, so i don't think a special handling for meilisearch is necessary

**Reply by tokubi** [reply]
Thanks, I'll try...
